understanding/20/error-identification.html

```diff
@@ -47,7 +47,24 @@ <h2>Intent of Error Identification</h2>
          <li>the user enters alphabetic characters or parentheses into their phone number field that only accepts numbers;</li>
          <li>the user enters a bid that is below the previous bid or the minimum bid increment.</li>
       </ul>
-
+<h3>Note on using native HTML form validation</h3>
+<p>With native HTML form validation, the use of the <code>required</code> attribute on a field has the consequence that 
```

Comment:
not just the `required` attribute. you can trigger browser-native form validation with other things like just `pattern`, `minlength`/`maxlength`, `min`/`max`